widgets.html

```diff
@@ -147,6 +153,35 @@ <h3>Installation</h3>
           With pip:
           {% highlight bash %}pip install nglview
 jupyter nbextension enable --py --sys-prefix nglview{% endhighlight %}
+        </div>
+        <div class="tab-pane" id="k3d">
+          <div class="jupyter-widget-header">
+            <span class="gallery-title">K3D-Jupyter</span>
+          </div>
+          <p>
+            <a href="https://github.com/K3D-tools/K3D-jupyter">K3D</a> is a Jupyter widget to 3-D plotting using WebGL.
```

Comment:
I think that sentence could be improved slightly. How about:

> K3D lets you create 3D plots backed by WebGL.

Reply:
Thanks. In next commit that will be changed.